install_config/install/deploy_router.adoc

```diff
@@ -234,39 +236,71 @@ router] on your OpenShift cluster using IP failover.
 
 === Customizing the Default Routing Subdomain
 
+You can customize the routing subdomain by
+either modifying the master configuration file,
+or by setting environment variables in the deployment configuration.
+
+Modifying the master configuration file::
 You can customize the suffix used as the default routing subdomain for your
 environment using the
 link:../../install_config/master_node_configuration.html#master-configuration-files[master
 configuration file] (the *_/etc/origin/master/master-config.yaml_* file by
 default). The following example shows how you can set the configured suffix to
 *v3.openshift.test*:
-
++
 .Master Configuration Snippet
 ====
-
 ----
 routingConfig:
   subdomain: v3.openshift.test
 ----
 ====
-
++
 [NOTE]
 ====
 This change requires a restart of the master if it is running.
 ====
-
++
 With the OpenShift master(s) running the above configuration, the
 link:../../architecture/core_concepts/routes.html#route-hostnames[generated host
 name] for the example of a host added to a namespace *mynamespace* would be:
-
++
 .Generated Host Name
 ====
-
 ----
 myroute-mynamespace.v3.openshift.test
 ----
 ====
 
+Setting environment variables in the deployment configuration::
+
+You can use the `oc env` command to set environment variables
+on the router's deployment configuration object.
+Two variables control if and how the subdomain is formed.
+Their influence depends on whether or not a route has a hostname.
```

Comment:
Hmm, subdomain formed sounds a bit weird ... maybe something akin to : `You can control the hostnames [route spec.host] fields that the router will allow/admit using the following environment variables: `  and then describe the 2 variables below.